packages/init/src/SliceMachineInitProcess.ts

```diff
@@ -137,7 +137,13 @@ export class SliceMachineInitProcess {
 				appName: pkg.name,
 				appVersion: pkg.version,
 			});
-			await setupSentry();
+			try {
+				await setupSentry();
+			} catch (error) {
```

Comment:
I'm just not sure what would happen if the setup fails, and we still call the trackError that will call Sentry.

Can we just to protect add a console.log like `console.log("Failed to setup Sentry")`, it would help us to know if someone copy-paste an init error in an issue on day.